CHANGES.rst

```diff
@@ -13,6 +13,11 @@
 * Removed leftover ``autoclose_stale_pull_request`` configuration item for
   the stale pull request script.
 
+* The ``pull_request_handler`` decorator now accepts a list of actions to
+  trigger on. By default, functions decorated ``pull_request_handler`` are
+  called with any of unlabeled, labeled, synchronize, opened, milestoned, and
+  demilestoned. [#XX]
```

Comment:
Update PR number. You might like https://github.com/astropy/astropy-procedures/blob/master/next_pr_number.py for future :)